File: Proyectos Python (3)/Valor a pagar mas iva.py
print("Vamos a determinar el valor total a pagar incluyendo el valor del iva")
cp=0
vu=0
t=0

cp=int(input("ingrese la cantidad de articulos: "))
for num in range(0,cp):
    vu=int(input("Ingrese el valor de cada producto: "))
    t=t+vu

iva=t*16/100
promedio=t+iva
print("El valor total a pagar es: ",promedio,"$")
print("El valor del 16% de iva incluido es: ",iva)

File: Proyectos C++ (3)/Valor a pagar mas iva.cpp
#include<iostream>

using namespace std;

int main(){

	int i, cp, vu, acum, prom, prn;
	cout<<"Programa que va a determinar el total a pagar y el valor del iva"<<endl;
		cout<<"Ingrese la cantidad de productos: "<<endl ;
		cin>>cp;
		acum=0;
	for(i=1;i<=cp;i++)
	{
	
		cout<<"Ingrese el valor de cada producto: "<<endl ;
		cin>>vu;
		acum=acum+vu;
		
	}
	prn=acum+(acum*16/100);
	prom=acum+cp;
	cout<<"El valor total a pagar es: " <<prn<<endl;
	cout<<"El valor del iva a incluir es: "<<acum*16/100;

	
	return 0;

}	
